shell-completion/bash/systemd-confext

```diff
@@ -64,6 +65,9 @@ _systemd-confext() {
             --image-policy)
                 comps=''
                 ;;
+            --mutable)
+                comps='no yes auto import ephemeral ephemeral-import'
```

Comment:
Please make the command support `--mutable=help` then use it here.
See e.g. `help_dns_types()` in src/resolve/resolvectl.c .